go/test/endtoend/onlineddl/vrepl_stress/onlineddl_vrepl_mini_stress_test.go

```diff
@@ -383,9 +385,10 @@ func checkTablesCount(t *testing.T, tablet *cluster.Vttablet, showTableName stri
 		select {
```

Comment:
alternatively we could go with:

```go
for {
    // all the original loop before the select ...

    select {
    case <-time.After(time.Second):
        continue
    case <-ctx.Done():
    }
    break
}

assert.Equal(...)
```

Reply:
Let's use this approach. We generally do not use labels in the vitess codebase -- not sure that there's any decision per-se, but we just don't use them, so let's continue not using them.

Reply:
sgtm!